lib/xcode.js

```diff
@@ -109,7 +109,8 @@ async function getVersionWithoutRetry () {
 
   // we want to read the CFBundleShortVersionString from Xcode's plist.
   // It should be in /[root]/XCode.app/Contents/
-  const plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  let plistPath = xcodePath.replace(XCODE_SUBDIR, "/Contents/Info.plist");
+  plistPath = plistPath.replace(new RegExp("/$"), "").trim();
```

Comment:
`plistPath.replace(new RegExp("/$"), "").trim();` is a direct copy and paste from existing code in appium-xcode. maybe the entire file should be rewritten to use resolve however it's out of scope for this bug fix.